src/cli/cli_output.cpp

```diff
@@ -111,11 +111,15 @@ void Output::OutputNewLine(void)
 
 void Output::OutputSpaces(uint8_t aCount)
 {
-    char format[sizeof("%256s")];
-
-    snprintf(format, sizeof(format), "%%%us", aCount);
+    for (; aCount >= 4; aCount -= 4)
+    {
+        OutputFormat("    ");
+    }
 
-    OutputFormat(format, "");
+    for (; aCount != 0; aCount--)
+    {
+        OutputFormat(" ");
+    }
```

Comment:
Is it possible to use a single `OutputFormat("%*s", aCount, "")`

Reply:
Looking the referred PR issue, I think `gnu_printf` is warning about format string is empty, not that arguments is empty.

Or at lease we can use this:

```
if (aCount > 0)
    OutputFormat("%*s", aCount - 1, " ")
```

Reply:
You are right. It's about format string being empty. Thanks.
Changed it in new push to `OutputFormat("%*s", aCount, "")`. I agree. better to have a single line. 

This was discovered by `matter` code updating to latest OT stack. We can change it  if their CI/toolchain still gives a warning on empty string.

Reply:
Thanks.